.cirrus.yml

```diff
@@ -56,7 +56,10 @@ task:
   << : *GLOBAL_TASK_TEMPLATE
   container:
     image: ubuntu:focal
+    cpu: 4  # Double CPU and Memory to avoid timeout
+    memory: 16G
   env:
+    MAKEJOBS: "-j8"
```

Comment:
`cpu: 4`, `MAKEJOBS: "-j8"` and `memory: 16G` are interconnected. Is there any justification that that combination is optimal?

Reply:
MAKEJOBS is also used to run the functional tests in ci in parallel. To use maximum CPU during functional tests, at least num_cpu * 2 needs to be passed.

```
$ ./test/functional/test_runner.py --help|grep parallel
  --jobs JOBS, -j JOBS  how many test scripts to run in parallel. Default=4.

Reply:
> To use maximum CPU during functional tests, at least num_cpu * 2 needs to be passed.

TIL.

Reply:
At least this is what I found out a while back. Might have changed in the meantime